x-pack/solutions/security/plugins/security_solution/server/lib/siem_migrations/rules/task/agent/nodes/match_prebuilt_rule/match_prebuilt_rule.ts

```diff
@@ -74,8 +74,11 @@ export const getMatchPrebuiltRuleNode = ({
           elastic_rule: {
             title: matchedRule.name,
             description: matchedRule.description,
-            id: matchedRule.installed_rule_id,
             prebuilt_rule_id: matchedRule.rule_id,
+            id: matchedRule.current?.id,
+            integration_ids: matchedRule.target?.related_integrations?.map((i) => i.package),
+            severity: matchedRule.target?.severity,
```

Comment:
should we set some default value incase `matchedRule.target?.severity == undefind`? Same with `risk_score`. Or this should not be possible?

Reply:
hmmm good point. All prebuilt rules must have a value for `severity` and `risk_score`, but technically the `target` entry can be undefined. So yes, you are right.
I added the default values here  https://github.com/elastic/kibana/pull/206833/commits/4e627a6e9af1f700c70e61c6b990c699410d7253